requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/PennyLaneAI/pennylane.git@master
+pennylane>=0.15
```

Comment:
This can be reverted as the test case modified in #16 doesn't depend on PennyLane v0.17.0 (the original reason for pinning to `master`).